AmplifyPlugins/Auth/AWSCognitoAuthPlugin/HubEvents/AuthHubEventHandler.swift

```diff
@@ -69,6 +73,13 @@ class AuthHubEventHandler: AuthHubEventBehavior {
                 }
                 self?.sendUserSignedOutEvent()
 
+            case HubPayload.EventName.Auth.deleteUserAPI:
+                guard let event = payload.data as? AWSAuthDeleteUserOperation.OperationResult,
+                    case .success = event else {
+                        return
+                }
+                self?.sendUserDeletedEvent()
```

Comment:
Regarding my previous comment above, it looks like we are sending both events? signedOut and userDeleted? Is this approved across platform?

Reply:
yes, see my previous comment.